dateutil/parser/_parser.py

```diff
@@ -324,19 +324,17 @@ def jump(self, name):
         return name.lower() in self._jump
 
     def weekday(self, name):
-        if len(name) >= min(len(n) for n in self._weekdays.keys()):
-            try:
-                return self._weekdays[name.lower()]
-            except KeyError:
-                pass
+        try:
```

Comment:
Is this part of the PoC or something you actually want merged?

Reply:
PoC to demonstrate the `asv continuous` comparison.  It should go into a separate PR-- it's a pretty nice perf improvement.

I'll revert this part in the next draft.